codeforpdx/dwellinglybackend
# Property multiple tenants

## What issue is this solving?
Closing Backend Issue [#236](https://github.com/codeforpdx/dwellingly-app/issues/236)

## Any helpful knowledge/context for the reviewer?
**Is a reseeding of the database necessary?** Nope!
**Any new dependencies to install?** Nope!
**Any special requirements to test?:** Yes! See below:
- Make sure that all tenants API routes work as expected
- Make sure that all property API routes work as expected
- Property Name should now exist on TenantModel
- An array of Tenants should now exist on PropertyModel

## Other comments:
I've opened issue [#254](https://github.com/codeforpdx/dwellingly-app/issues/254) to add two new attributes to the TenantModel: "Unit", and an "Added On" date.

## Please make sure you've attempted to meet the following coding standards
 **Code builds successfully** - CHECK!
**Code has been tested and does not produce errors** - I did manual testing here. Let me know if you want me to write tests for this.
**Code is readable and formatted** - Check! :)
**There isn't any unnecessary commented-out code** - nope!
